examples/main/layer-examples.js

```diff
@@ -17,9 +17,9 @@ import {
 
   ChoroplethLayer,
   ChoroplethLayer64,
-  ExtrudedChoroplethLayer64,
+  ExtrudedChoroplethLayer64
 
-  Container
+  // Container
```

Comment:
We're exporting the `get` function now separately. This could be nice to showcase as a way to get nested data.

Agree we should not use the Container class, it's being removed from the exports.